priv/static/js/app.js

```diff
@@ -0,0 +1,1483 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
priv/static/js/app.js is auto generated on every server build. should not be under git.